packages/migrate/src/__tests__/fixtures/existing-db-1-draft-1-change/prisma/schema.prisma

```diff
@@ -15,5 +15,5 @@ model Blog {
 
 model Cat {
   id    Int @id
-  lifes Int
+  lives Int
```

Comment:
I'm assuming this is purely test data and doesn't have any corresponding anything...